app/screens/send-bitcoin-screen/send-bitcoin-details-screen.tsx

```diff
@@ -558,15 +558,16 @@ const SendBitcoinDetailsScreen: React.FC<Props> = ({ route }) => {
   let LnUrlMinMaxAmount: React.ReactNode = null
 
   if (lnurlParams && convertMoneyAmount) {
+    const { min, max } = lnurlParams
     LnUrlMinMaxAmount = (
```

Comment:
This should only be shown if `canSetAmount` is true

Reply:
addressed here https://github.com/GaloyMoney/galoy-mobile/pull/1523/commits/aa5b944c59919282764b6c3e4f35c455027ada28